modules/monitoring-creating-cluster-monitoring-configmap.adoc

```diff
@@ -3,36 +3,55 @@
 // * monitoring/cluster_monitoring/configuring-the-monitoring-stack.adoc
 
 [id="creating-cluster-monitoring-configmap_{context}"]
-= Creating cluster monitoring ConfigMap
+= Creating a cluster monitoring ConfigMap
 
-To configure the Prometheus Cluster Monitoring stack, you must create the cluster monitoring ConfigMap.
+To configure the {product-title} monitoring stack, you must create a cluster monitoring ConfigMap.
 
 .Prerequisites
 
-* An installed `oc` CLI tool
-* Administrative privileges for the cluster
+* You have access to the cluster as a user with the cluster-admin role.
+* You have installed the OpenShift CLI (`oc`).
 
 .Procedure
 
-. Check whether the `cluster-monitoring-config` ConfigMap object exists:
+. Check whether a `cluster-monitoring-config` ConfigMap object exists:
 +
+[source,terminal]
 ----
 $ oc -n openshift-monitoring get configmap cluster-monitoring-config
 ----
 
-. If it does not exist, create it:
+. Configure the `data` section in the ConfigMap.
```

Comment:
@olga-mir on review, could this line read better as follows, since we are doing more than just configuring the data section in the first bullet point?:

`. Configure the ConfigMap.`